src/views/PartnersView.vue

```diff
@@ -88,9 +88,12 @@ import odrive from '@clubcapra/assets/media/partners/odrive.png';
         </div>
         <div class="col-md-12">
           <div class="mt-3 text-center">
+            <!--
```

Comment:
Don't leave any code as a comment, it's better to remove it completely. The code will still be available in the git history, and it will be easy to put it back. It's cleaner to remove it completely.